### PR TITLE
License: Update License file and Third Party Notices

### PR DESCRIPTION
**Pull request by obastemur**

### CODE REVIEW

File: THIRD-PARTY-NOTICES.txt [hunk 1]
```diff
@@ -5,69 +5,1132 @@ reserves all other rights not expressly granted, whether by
...
+18.	MakeNewWorld() function
+19.	MochiKit (https://github.com/mochi/mochikit)
+20.	Mozilla XML-RPC Client code
+21.	Octane (https://github.com/chromium/octane)
+22.	pdfjs.js (https://github.com/mozilla/pdf.js)
+23.	Pixastic Lib (https://github.com/jseidelin/pixastic)
+24.	Prototype (https://github.com/sstephenson/prototype)
+25.	Sputnik  (https://github.com/fholm/IronJS/tree/master/Src/Tests/Sputnik/sputnik-v1)
+26.	SunSpider
+27.	test262 (https://github.com/ljharb/test262/)
```
Comment (dilijev) [inline THIRD-PARTY-NOTICES.txt]: While we're at it, should we update the test262 URL to the new home at https://github.com/tc39/test262 ?

File: THIRD-PARTY-NOTICES.txt [hunk 2]
```diff
@@ -5,69 +5,1132 @@ reserves all other rights not expressly granted, whether by
...
+1.	backbone.suggestions (https://github.com/qloo/backbone.suggestions)
+2.	BeatDetektor.js
+3.	Catch
+4.	CRC.h
+5.	deltablue.js
+6.	Dojo Toolkit (https://dojotoolkit.org)
+7.	DSP.js (https://github.com/corbanbrook/dsp.js/)
+8.	ecma-proposal-object.getownpropertydescriptors (https://github.com/tc39/proposal-object-getownpropertydescriptors/)
+9.	File copyright Microsoft and Geoff Norton
+10.	File with code copyright Baron Schwartz
```
Comment (dilijev) [inline THIRD-PARTY-NOTICES.txt]: Should we be more specific about which file for 9 and 10?

File: THIRD-PARTY-NOTICES.txt (hunk 1 above)
Comment (obastemur) [inline THIRD-PARTY-NOTICES.txt]: good one. updated.

File: THIRD-PARTY-NOTICES.txt (hunk 2 above)
Comment (obastemur) [inline THIRD-PARTY-NOTICES.txt]: First is partial but second, I have updated.

**Comment by obastemur** [comment]
@dotnet-bot test Ubuntu shared_ubuntu_linux_debug please

**Comment by obastemur** [comment]
@dotnet-bot test Ubuntu shared_ubuntu_linux_debug please

**Comment by mmitche** [comment]
@dotnet-bot test Ubuntu shared_ubuntu_linux_debug

**Comment by mmitche** [comment]
@obastemur Ubuntu shared_ubuntu_linux_debug is for a PR only targeting master. The reason it showed up here is most likely that the hash of this PR matched the hash of another PR against master.  When the other PR asked GitHub to put a status check for hash abcd, the check showed up in two places.

If you push another ref here, it should disappear.

**Comment by mmitche** [comment]
(or you could just ignore it)

**Comment by obastemur** [comment]
@mmitche Thanks! for instant response. I will try.

**Comment by obastemur** [comment]
This PR was previously approvedby legal d. @dilijev thanks for the additional review and suggestions.

**Pull request merged**